demo/meeting/src/containers/LocalMediaStreamMetrics/index.tsx

```diff
@@ -0,0 +1,176 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+import {
+  useMediaStreamMetrics,
+  useAudioVideo,
+  PopOverHeader
+} from 'amazon-chime-sdk-component-library-react';
+
+import MediaStatsList from '../../components/MediaStatsList/index';
+import MetricItem from '../../components/MediaStatsList/MetricItem';
+import { StyledMediaMetricsWrapper } from '../../components/MediaStatsList/Styled';
+
+export const LocalMediaStreamMetrics: React.FC = () => {
+  const audioVideo = useAudioVideo();
+  const localAttendeeId = (audioVideo as any).audioVideoController
+    ?.realtimeController?.state?.localAttendeeId;
+  const {
+    audioPacketsSentFractionLossPercent,
+    audioPacketsReceivedFractionLossPercent,
+    availableIncomingBandwidth,
+    availableOutgoingBandwidth,
+    videoStreamMetrics
+  } = useMediaStreamMetrics();
+  const localVideoStreamMetrics = videoStreamMetrics[localAttendeeId];
+  const ssrcArray = localVideoStreamMetrics
+    ? Object.keys(localVideoStreamMetrics)
+    : [];
+  const isLocalAudioActive =
+    audioVideo &&
+    audioPacketsSentFractionLossPercent !== null &&
+    audioPacketsReceivedFractionLossPercent !== null;
+  const isLocalVideoActive = audioVideo && ssrcArray.length !== 0;
+  const hasBandwidthInfo =
+    availableIncomingBandwidth !== null && availableOutgoingBandwidth !== null;
+  return (
+    <StyledMediaMetricsWrapper>
+      {isLocalAudioActive && (
+        <>
+          <PopOverHeader title={'Audio statistics'} />
+          <MediaStatsList>
+            <MetricItem metricName="" metricValues={['Up', 'Down']} />
+            <MetricItem
+              metricName="1s Loss"
+              metricValues={[
+                audioPacketsSentFractionLossPercent !== null ? audioPacketsSentFractionLossPercent.toString() : "",
+                audioPacketsReceivedFractionLossPercent !== null ? audioPacketsReceivedFractionLossPercent.toString(): ""
+              ]}
+            />
+          </MediaStatsList>
+        </>
+      )}
+      {isLocalVideoActive && (
+        <>
+          <PopOverHeader title={'Video statistics'} />
+          <MediaStatsList>
+            <MetricItem
+              metricName="Bit rate (kbps)"
+              metricValues={
+                ssrcArray &&
+                ssrcArray.map(ssrc => {
```

Comment:
I think `srcArray.map` should be sufficient, because `srcArray` would be `[]` at the least and `isLocalVideoActive` will fail so it will not reach here right?

Reply:
Good point. Will remove it.